editor/editor_node.cpp

```diff
@@ -6334,6 +6355,22 @@ void EditorNode::reload_instances_with_path_in_edited_scenes() {
 				}
 			}
 
+			// Restore the scene's editable instance and folded states.
+			Ref<SceneState> ss_inst = owner->get_scene_instance_state();
```

Comment:
Hmm...I looked back over this code, and yeah, I couldn't actually find a situation where updating the scene state during this reimport was ever used or required. I can't remember if it was ever used, but it may very well be a holdover from an earlier iteration of the system. As such, I've taken it out as I was not able to recreate a situation where it was ever required.